visualization_scripts/run.py

```diff
@@ -0,0 +1,145 @@
+""" 
+script loads style file and TPS output file into Cytoscape session
+
+@author: Adam Shedivy
+
+"""
+
+# imports 
+from py2cytoscape.data.cynetwork import CyNetwork
+from py2cytoscape.data.cyrest_client import CyRestClient
+from py2cytoscape.data.style import StyleUtil
+import py2cytoscape.util.cytoscapejs as cyjs
+import py2cytoscape.cytoscapejs as renderer
+from py2cytoscape import cyrest
+import networkx as nx
+import pandas as pd
+import json
+import sys
+import time
+import subprocess
+import os
+
+#assert Cytoscape is running on machine
+
+def process_exists(process_name):
+    """
+    function to assert Cytoscape is running
+
+    Args:
+        process_name (str): name of process to check for
+        example - 'Cytoscape.exe'
+
+    Returns:
+        true if process runnning, else false
+
+    """
+    bytes_name = str.encode(process_name)
+    call = 'TASKLIST', '/FI', 'imagename eq %s' % process_name
```

Comment:
Is this the Windows equivalent to the `ps` command?

Reply:
Yes. the tasklist command lets you see the running processes on your machine. You can specify certain criteria such as memory usage, run time, image file name, etc. If I launch Cytoscape and call `tasklist /fi "imagename eq Cytoscape.exe"` from the command prompt the output is:
```
Image Name                     PID Session Name        Session#    Mem Usage
========================= ======== ================ =========== ============
Cytoscape.exe                22364 Console                    1    243,320 K
```